cmd/resources/resources.go

```diff
@@ -352,10 +356,49 @@ func NewOperationCmd(parentCmd *cobra.Command, client resources.Client, op Opera
 		Use:   op.Use,
 	}
 
+	cmd.SetUsageTemplate(operationUsageTemplate())
+
 	opCmd.cmd = cmd
 	_ = opCmd.initFlags()
 
 	parentCmd.AddCommand(cmd)
 
 	return cmd
 }
+
+func operationUsageTemplate() string {
+	return fmt.Sprint(`Usage:{{if .Runnable}}
+  {{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}
+  {{.CommandPath}} [command]{{end}}{{if gt (len .Aliases) 0}}
+
+Aliases:
+  {{.NameAndAliases}}{{end}}{{if .HasExample}}
+
+Examples:
+{{.Example}}{{end}}{{if .HasAvailableSubCommands}}{{$cmds := .Commands}}{{if eq (len .Groups) 0}}
+
+Available Commands:{{range $cmds}}{{if (or .IsAvailableCommand (eq .Name "help"))}}
+  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{else}}{{range $group := .Groups}}
+
+{{.Title}}{{range $cmds}}{{if (and (eq .GroupID $group.ID) (or .IsAvailableCommand (eq .Name "help")))}}
+  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{if not .AllChildCommandsHaveGroup}}
+
+Additional Commands:{{range $cmds}}{{if (and (eq .GroupID "") (or .IsAvailableCommand (eq .Name "help")))}}
+  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{end}}{{end}}{{if .HasAvailableLocalFlags}}
+
+Required flags:
+{{WrappedRequiredFlagUsages . | trimTrailingWhitespaces}}
+
+Optional flags:
+{{WrappedOptionalFlagUsages . | trimTrailingWhitespaces}}
+
+Global Flags:
+{{rpad "  -h, --help" 29}} Help for this command
```

Comment:
it felt weird to include this within the "optional" flags, even though it's not technically a global flag it kinda is? lmk if we want to move this somewhere else